### PR TITLE
mail/postfix: added sender / recipient access maps

### PR DESCRIPTION
**Pull request by mimugmail**
- removed from general 
- 2 new menu items
- functionality to reject or allow addresses

### CODE REVIEW

File: mail/postfix/src/opnsense/mvc/app/controllers/OPNsense/Postfix/RecipientController.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+
+namespace OPNsense\Postfix;
+
+class RecipientController extends \OPNsense\Base\IndexController
+{
+    public function indexAction()
+    {
+        $this->view->title = gettext("Postfix Recipients");
```
Comment (fichtner) [inline mail/postfix/src/opnsense/mvc/app/controllers/OPNsense/Postfix/RecipientController.php]: no more titles please unless there is a purpose (it should auto-regenerate from Menu.xml)

File: mail/postfix/src/opnsense/mvc/app/controllers/OPNsense/Postfix/SenderController.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+
+namespace OPNsense\Postfix;
+
+class SenderController extends \OPNsense\Base\IndexController
+{
+    public function indexAction()
+    {
+        $this->view->title = gettext("Postfix Senders");
```
Comment (fichtner) [inline mail/postfix/src/opnsense/mvc/app/controllers/OPNsense/Postfix/SenderController.php]: same here

File: mail/postfix/src/opnsense/mvc/app/models/OPNsense/Postfix/Menu/Menu.xml [hunk 3]
```diff
@@ -3,6 +3,8 @@
     <Postfix cssClass="fa fa-envelope fa-fw">
       <General url="/ui/postfix/general/index" order="10"/>
       <Domains url="/ui/postfix/domain/index" order="20"/>
+      <Recipient url="/ui/postfix/recipient/index" order="30"/>
```
Comment (fichtner) [inline mail/postfix/src/opnsense/mvc/app/models/OPNsense/Postfix/Menu/Menu.xml]: For consistency this should be "Recipients" and "Senders"

**Comment by mimugmail** [comment]
Fixed :)

**Comment by fichtner** [comment]
Ready for pull?

**Comment by fabianfrz** [comment]
@fichtner on my machine it looks like it is ;)

**Comment by mimugmail** [comment]
Yes, please pull :)

Am 17.12.2017 9:52 nachm. schrieb "Fabian Franz, BSc." <
notifications@github.com>:

**Comment by fichtner** [comment]
Merged, thanks!

**Comment by mimugmail** [comment]
@fabianfrz @fichtner If I add a third OptionField requiring an argument via TextField, does that require some JS foo to only be displayed when it is choosen? Or is there some method I don't know of :)

**Comment by fichtner** [comment]
I think Tor plugin has such a case, but we simply never hide the optional textfield... It's always validated but only used if the other option is set. Acme-client does more advanced things in that regard, it's somehow possible but I don't know how.

**Pull request merged**